backend/node_modules/.bin/mime

```diff
@@ -2,14 +2,11 @@
 basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
```

Comment:
Remove `node_modules` and add it to `.gitignore` so that they are not available for commit